### PR TITLE
Login Page

### LINKED ISSUE #3: Add login page

### PR DESCRIPTION
**Pull request by n-aggarwal**

### CODE REVIEW

**Comment by n-aggarwal** [comment]
There is a slight issue with backend and how it should connect to the frontend, but apart from that everything works.

**Comment by n-aggarwal** [comment]
If you figure out how to connect the frontend to the backend, let me know. For testing main page, comment the axios, and use only the code in the `if(response.data.success)` block.

**Review by ananafrida — APPROVED** [review]

File: views/login.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  const [username, setUsername] = useState("");
+
+  const onRegisterClick = () => {
+    navigation.navigate("Register");
+  };
+
+  const onLogin = () => {
+    axios
+      .post(
+        "http://localhost:80/index.php/user/login",
```
Comment (ananafrida) [inline views/login.js:32]: @n-aggarwal try with your ip address here

**Comment by ananafrida** [comment]
> There is a slight issue with backend and how it should connect to the frontend, but apart from that everything works.

Look at slide 33 of class lecture. It should work.

**Comment by n-aggarwal** [comment]
Will do. Thanks!

**Pull request merged**